### DISCUSSION TITLE
error after docker installation

### DISCUSSION THREAD

**Discussion by golftiger13** [meltaxa/jukebox-portal#14] [Q&A] [answered]
after I started the container I received two error messages:
First
/usr/local/lib/python3.9/site-packages/flask/app.py:2171: UserWarning: Current server name 'XXX.XXX.XX.XX:5000' doesn't match configured server name 'localhost:5000'

Second
libusb: error [_open_sysfs_attr] open /sys/bus/usb/devices  p /bConfigurationValue failed ret=-1 errno=2

**Comment by meltaxa** [comment, marked as answer]
Musicfig is reading a config.yml file that has the default SERVER_NAME as localhost:5000 instead of your ip address:port number. Ensure Docker is mounting the directory containing your config.yml file and the SERVER_NAME is set correctly.

Docker --device USB device should match the bus and device numbers from the command:
```
lsusb | grep 0e6f:0241
```
Can you paste the lsusb result and Docker run command?

**Reply by golftiger13** [reply]
I am so sorry, I didn't check the config file....and now it works.....great fine thanks.
The Skylander figures are working, only some of them have issue. It seems they have two nfc's.
And one has a NFC tag only digits and can not be identified, unfortunately it is my favourite figure.
